app/scripts/modules/amazon/src/serverGroup/details/resize/resizeCapacity.component.html

```diff
@@ -85,4 +95,15 @@
         instances
     </div>
   </div>
+  <div class="row" ng-if="$ctrl.command.min == $ctrl.command.max && $ctrl.serverGroup.scalingPolicies.length > 0">
+    <div class="col-md-7 col-md-offset-3">
+      <div class="well-compact alert alert-warning">
+        <b>Warning</b>: this server group has
+        <span ng-if="$ctrl.serverGroup.scalingPolicies.length == 1">a scaling policy.</span>
+        <span ng-if="$ctrl.serverGroup.scalingPolicies.length > 1">scaling policies.</span>
+        Scaling policies will not take effect in Simple Mode.
+        Switch to <a href ng-click="$ctrl.command.advancedMode = true">Advanced Mode</a>.
```

Comment:
It looks like these two lines are the only difference between the block above. Could this whole block be moved to the end of the file, and just wrap these two lines in an `ng-if`?

(I know it's not a ton of duplicate code, but it still might be nice to consolidate the shared parts)

Reply:
@christopherthielen and I originally made the opposite call to avoid duplicating the `advancedMode` logic prior to some of the later additions to the markup, but in its final state I totally agree that consolidating them is ideal.

Reply:
@anotherchrisberry pushed.